pkg/agent/testdata/AKSUbuntu1804+Containerd+IPMasqAgent/CustomData

```diff
@@ -126,6 +126,11 @@ write_files:
                 "subnet": "{{.PodCIDR}}",
                 "routes": [{ "dst": "0.0.0.0/0" }]
             }
+          },
+          {
+            "type": "portmap",
```

Comment:
is this intended?

Reply:
Somebody else forgot to `make generate` the code and merged the code ignoring the "generated code check".